examples/python/interceptors/default_value/greeter_client.py

```diff
@@ -0,0 +1,37 @@
+# Copyright 2015 gRPC authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""The Python implementation of the GRPC helloworld.Greeter client."""
+
+from __future__ import print_function
+
+import grpc
+
+import helloworld_pb2
+import helloworld_pb2_grpc
+from default_value_client_interceptor import DefaultValueClientInterceptor as _DefaultValueClientInterceptor
```

Comment:
[Import only packages and modules (almost always modules!); never classes, functions, or other elements of modules.](https://google.github.io/styleguide/pyguide.html?showone=Imports#Imports)